### ISSUE TITLE
Consider changing the license to the Hippocratic License

### ISSUE THREAD

**Issue by kimadactyl** [geeksforsocialchange/PlaceCal#1106]
https://firstdonoharm.dev/

**Comment by kimadactyl** [comment]
Might be worth revisiting when the license builder is done 

> The HL3 license builder with optional modules will be available for use in early 2022.

**Comment by kimadactyl** [comment]
Yeah I think there's a space for more meta tickets about the function of the repo tho?

We can come back to it when we start this milestone

**Comment by katjam** [comment]
Not sure where this lies in terms of priority.

**Comment by kimadactyl** [comment]
Probably something to think about before the v1.0 launch but no hurry
